README.md

```diff
@@ -1514,3 +1514,74 @@ Here is an example of how to use the option C syntax:{import { default as myAlia
 
 </p>
 </details>
+
+##### 38.  for Inheritance, Subclassing and Extending built in class (Level: Hard)
+
+```javascript
+function Animal(name) {
+    this.name = name;
+  }
+  
+  Animal.prototype.eat = function() {
+    console.log(this.name + " is eating.");
+  };
+  
+
+  function Dog(name) {
+    Animal.call(this, name); 
+  }
+  
+  Dog.prototype = Object.create(Animal.prototype); 
+  Dog.prototype.constructor = Dog; 
+  
+  Dog.prototype.bark = function() {
+    console.log(this.name + " is barking.");
+  };
+  
+
+  function CustomArray() {
+    Array.call(this);
+  }
+  
+  CustomArray.prototype = Object.create(Array.prototype);
+  CustomArray.prototype.constructor = CustomArray;
+  
+  CustomArray.prototype.sum = function() {
+    return this.reduce((acc, val) => acc + val, 0);
+  };
+  
+  var dog = new Dog("Buddy");
+  dog.eat();
+  dog.bark();
+  
+  var numbers = new CustomArray();
+  numbers.push(1, 2, 3, 4, 5);
+  console.log(numbers.sum()); 
+  
+```
+
+What will be the output of the following code
+
+- A: Buddy is eating , Buddy is barking ,  15
```

Comment:
missing period in the options. 

actual output: Buddy is eating. Buddy is barking. 15

So, please add "." here or remove it from the code